### ISSUE TITLE
Update README.md

### ISSUE THREAD

**Issue by edan-bainglass** [aiidateam/aiida-workgraph#181]
The README.md examples do not reflect the current state of the docs and suggest a different use of the work graph.

**Comment by superstar54** [comment]
@edan-bainglass suggests using `task.calcfunction` instead of asking users to import `calcfunction` from the `aida.engine`.

So update the README to
```python
# define add calcfunction
@task.calcfunction
def add(x, y):
    return x + y

```

**Comment by edan-bainglass** [comment]
Indeed. The fact that AiiDA's mechanics are supported is a neat feature. But at the end of the day, the real gain from aiida-workgraph is that one doesn't need to know about AiiDA (e.g. no AiiDA imports necessary).

**Comment by superstar54** [comment]
The README is updated using the latest API (`@task` and `@task.graph`) in https://github.com/aiidateam/aiida-workgraph/pull/671, so I close this issue.
